core/db/entities.py

```diff
@@ -149,6 +149,9 @@ class Procurement(db.Model):
     start_date = sqla.Column(sqla.DateTime(), nullable=False)
     end_date = sqla.Column(sqla.DateTime(), nullable=False)
     status = sqla.Column(sqla.Enum(const.StatusEnum, name="procurement_plan_status"), nullable=False)
+    procurement_status = sqla.Column(
+        sqla.Enum(const.ProcurementStatusEnum, name="procurement_plan_status"), nullable=False
```

Comment:
I think we should give this a unique name (it's currently set to same as the other "status")

Reply:
Fixed now